.github/workflows/lint.yml

```diff
@@ -10,9 +10,20 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: YAML Lint
+      - run: mkdir -p super-linter.report
+      - name: Lint Python
         uses: github/super-linter@v3
         env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           VALIDATE_ALL_CODEBASE: false
-          VALIDATE_YAML: true
           VALIDATE_PYTHON_BLACK: true
+          VALIDATE_PYTHON_FLAKE8: true
+          PYTHON_FLAKE8_CONFIG_FILE: setup.cfg
+          VALIDATE_PYTHON_ISORT: true
+          PYTHON_ISORT_CONFIG_FILE: setup.cfg
+      - name: Lint YAML
```

Comment:
Both the python version and the yaml version can be the same task.

Reply:
Merged.  I did as suggested but...

My sense was that contributors would want to know-at-a-glance errors were raised in C/C++, Python, shell, or YAML.  Lumping all languages into a single task forces contributors to scan thru all logs in search of errors instead of only the logs for the language that failed.

Reply:
I buy that, but really you should be able to go to the review or check tab and see all the annotations inline for the found issues.

Let's see how this works out in practice. I appreciate your work on cleaning up the code.